crates/ra_syntax/Cargo.toml

```diff
@@ -8,7 +8,7 @@ description = "Comment and whitespace preserving parser for the Rust langauge"
 repository = "https://github.com/rust-analyzer/rust-analyzer"
 
 [dependencies]
-unicode-xid = "0.1.0"
```

Comment:
I think we no longer need Unicode-xid at all, could you try removing this line?

Reply:
Done